reporter/datadog_reporter.go

```diff
@@ -669,7 +669,7 @@ func (r *DatadogReporter) getPprofProfile() {
 
 func createTags(userTags Tags, runtimeTag, version string) Tags {
 	tags := append(Tags{}, userTags...)
-	customAttributes := []string{"container_id", "container_name", "thread_name", "pod_name"}
+	customAttributes := []string{"container_id", "container_name", "pod_name"}
```

Comment:
We need to make sure that backend knows about this attributes both for native and ebpf family.